zk/README.md

```diff
@@ -30,6 +30,59 @@ To configure this check for an Agent running on a host:
 
 2. [Restart the Agent][6].
 
+#### Enabling SSL
+
+Zookeeper 3.5 introduced the ability to use SSL authentication. The agent is able to connect to Zookeeper using SSL as well.
+
+One thing to note is that Zookeeper uses `truststores` and `keystores` JKS files while the Agent TLS/SSL configuration uses `tls_ca_cert`, `tls_cert`, and `tls_private_key` config options that accept PEM files.
+It may be confusing at first for those familiar with the JKS files to map to PEM files.
+
+It is assumed that one already has the server and client truststore and keystore files to start Zookeeper in SSL mode in the first place.
+More information on setting up Zookeeper in SSL mode first can be found [here][13].
+
+We will use following files as examples:
+- `server_truststore.jks`
+- `server_keystore.jks` 
+- `client_truststore.jks`
+- `client_keystore.jks`
+
+We also assume that both sides' `keystore`s and `truststore`s have each other's certificates, meaning that a Java Zookeeper client can already connect to a Zookeeper server.
+
+
+_We first will get `cert.pem` file from `client_truststore.jks`._
+1. Use `keytool` to export `client_truststore.jks` to a PKCS12 file called `cert.p12`:
+    ```
+    keytool -importkeystore -srckeystore client_truststore.jks -destkeystore cert.p12 -srcstoretype jks -deststoretype pkcs12
+    ```
+2. Use `openssl` to export the new PKCS12 `cert.p12` file to `cert.pem` PEM file:
+    ```
+    openssl pkcs12 -in cert.p12 -out cert.pem
+    ```
+
+_Next we will get `ca_cert.pem` file from `server_truststore.jks`._
```

Comment:
The server_truststore contains the certificate of the client that is trusted. This maps to `cert.pem`, i.e the cert of the client.